src/server/views/index.jade

```diff
@@ -0,0 +1,5 @@
+extends layout
```

Comment:
Yes, but right now, none of these views are being used. I vote for removing them, they're trivial to add back in once we're ready for custom pages